src/app/views/templates/sms-verification-warning.view.html

```diff
@@ -17,6 +17,7 @@
         Your account can use <%= numAvailable %> more free verifications until
         free SMS OTP verification is automatically disabled for all owned forms.
```

Comment:
I think we should just wait for sonjia for the copywriting lol! Here you put disabled for all owned forms. Then below you put Forms with Twilio not affected. Very confusing sentences.

Reply:
this is her copy writing HAHAHA cos ytd i realized the same thing you did and asked sonjia to update the copy but i forgot to update after she updated. 

only the modal's copywriting isn't approved by sonjia

Reply:
Please double check this! Reads awkwardly